### ISSUE TITLE
Disable double click listener for faster identification of clicks

### ISSUE THREAD

**Issue by wizarlock** [saket/telephoto#116]
When I click on an image, there is a long response (about a second), I need it faster. I think this is because of the processing onDoubleClick. Can I somehow remove onDoubleClick? Or is it not because of the processing onDoubleClick? I try to disable the processing with onDoubleClick = { _, _ -> }, but it will still be processed

**Comment by saket** [comment]
Yeah, telephoto expects a `onDoubleClick` listener to be always present. There may be a workaround: you could prevent telephoto from waiting for double taps by overriding its view configuration:

https://github.com/saket/telephoto/blob/03ec023e24565cb601a2bc5e3e5e985a22973b90/zoomable/src/commonMain/kotlin/me/saket/telephoto/zoomable/internal/tappableAndQuickZoomable.kt#L273-L275

```kotlin
val foo = object : ViewConfiguration by LocalViewConfiguration.current {
  override val doubleTapTimeoutMillis: Long get() = 0
}
CompositionLocalProvider(LocalViewConfiguration provides foo) {
  ZoomableAsyncImage(…)
}
```

**Comment by saket** [comment]
Fixed by [17b553b: Make double click listener nullable ](https://github.com/saket/telephoto/commit/17b553be47371cfbf698695652a82356e8583e25).

**Comment by saket** [comment]
Released as part of 0.15.0: https://github.com/saket/telephoto/releases/tag/0.15.0
